abdulfatai360/banka
# #165653750 Implement feedback on frontend

#### What does this PR do?
Implement frontend feedback

#### Description of Task to be completed?
- Put app logo on a single line
- Change placeholders for first name and last name fields
- On signin form, change the anchor text for the link that leads to the signup for to 'Signup'
- Add confirmation dialog box when a staff wants to delete an account
- Use icons to replace some action buttons

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/55/head:ch-implement-frontend-feedback-165653750`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-implement-frontend-feedback-165653750`
- Your directory will now be an exact copy of the PR

#### What are the relevant pivotal tracker stories?
[#165653750](https://www.pivotaltracker.com/story/show/165653750)
